packages/trpc/server/routers/viewer/admin/setSMSLockState.handler.ts

```diff
@@ -16,17 +16,20 @@ type GetOptions = {
 const setSMSLockState = async ({ input }: GetOptions) => {
   const { userId, username, teamId, teamSlug, lock } = input;
   if (userId) {
+    const userToUpdate = await prisma.user.findUnique({ where: { id: userId } });
+    if (!userToUpdate) throw new TRPCError({ code: "BAD_REQUEST", message: "User not found" });
+    const userUpdateInput = {
+      smsLockState: lock ? SMSLockState.LOCKED : SMSLockState.UNLOCKED,
+      ...(userToUpdate.smsLockState === SMSLockState.REVIEW_NEEDED && !lock
+        ? { smsLockReviewedByAdmin: true }
+        : {}),
+    };
```

Comment:
This is unnecessary. setSMSLockState is a privileged endpoint only accessible by admin, so you just have to update the reviewedByAdmin to true. 

smsLockReviewedByAdmin means the LockState whether Reviewe_Needed, unlocked locked etc is being done by admin, so we will just mark it true no matter the state.

Reply:
Done.